components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -1040,6 +1043,13 @@ def jvmcfg(self, args, config):
                 sb += " # %s" % settings
             self.ctx.out("%s=%s" % (k, sb))
 
+    def _glacier2_icessl_xml(self, config_props):
+        # Convert omero.glacier2.IceSSL.* properties to IceSSL.*
+        glacier2_icessl = [(k[15:], v) for (k, v) in config_props.items()
+                           if k.startswith('omero.glacier2.IceSSL.')]
```

Comment:
Nice strategy. If similar needs arise, I could see having this generalized for other prefixes.